string chess_square_pick
string chess_square_place
---
bool result